### PR TITLE
Change Lumi test to a separated 2 electron case to test cluster splitting

### PR DESCRIPTION
**Pull request by dhevang**
### Briefly, what does this PR introduce?

Use a G4 gps macro to configure a 2 electron event to test cluster splitting

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [x ] Other: Change to CI test

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

File: src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.mac [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# source 1
```
Comment (veprbl) [inline src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.mac:1]: If this doesn't work, please delete.

File: src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.hepmc [hunk 2]
```diff
@@ -0,0 +1,1203 @@
+HepMC::Version 3.02.05
```
Comment (veprbl) [inline src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.hepmc:1]: Please provide the original script instead, you should be able to run python/hepmc3 on the CI. Doing it in the same step is okay for now.

Comment (dhevang) [reply src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.hepmc:1]: I have a clunky cpp file to generate these hepmc events...
I wouldn't upload that.  Do you have a sample python script that I use as a template?

Comment (veprbl) [reply src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.hepmc:1]:
```
import sys

import numpy as np
from pyHepMC3 import HepMC3

seed = 42
rng = np.random.default_rng(seed)

skip = int(sys.argv[1]) * 1000

p = 1 # GeV

writer = HepMC3.WriterAscii("test.hepmc")

for ix in range(skip + 1000):
    costheta = -1 + 2 * rng.random()
    phi = 2 * np.pi * rng.random()
    if ix < skip:
        continue

    pt = np.sqrt(1 - costheta**2) * p
    px = pt * np.cos(phi)
    py = pt * np.sin(phi)
    pz = costheta * p
    mass = 0.511e-3
    E = np.sqrt(mass**2 + p**2)
    particle = HepMC3.GenParticle(HepMC3.FourVector(px, py, pz, E))
    particle.set_pdg_id(0)
    particle.set_status(1)

    particle_in1 = HepMC3.GenParticle(HepMC3.FourVector(0, 0, 10, 10))
    particle_in1.set_pdg_id(11)
    particle_in1.set_status(4)
    particle_in2 = HepMC3.GenParticle(HepMC3.FourVector(0, 0, -100, 100))
    particle_in2.set_pdg_id(2212)
    particle_in2.set_status(4)

    vertex = HepMC3.GenVertex(HepMC3.FourVector(0., 0., 0., 0.))
    vertex.add_particle_in(particle_in1)
    vertex.add_particle_in(particle_in2)
    vertex.add_particle_out(particle)

    event = HepMC3.GenEvent(HepMC3.Units.GEV, HepMC3.Units.MM)
    event.add_vertex(vertex)
    event.set_beam_particles(particle_in1, particle_in2)

    writer.write_event(event)
```

File: src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.mac (hunk 1 above)
Comment (veprbl) [reply src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.mac:1]: Yes, delete the file.

**Comment by veprbl** [comment]
It would seem like our test prints out an error, yet we don't report a failed status:
https://github.com/eic/EICrecon/actions/runs/4996729284/jobs/8950662526?pr=663#step:5:504
Not an issue to be resolved right away, but must be done at some point.

**Review by veprbl — APPROVED** [review]
LGTM

**Pull request merged**